### PR TITLE
update to the docs

### PR DESCRIPTION
**Pull request by rawls238**
@eytan Add a line about short-circuiting exposure logging and changes to language around the JS implementation

### CODE REVIEW

**Comment by eytan** [comment]
Thanks!

File: docs/04-logging.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ class Exp1(SimpleExperiment):
     return e
```
Comment (eytan) [inline docs/04-logging.md]: Hrm, where does this `return e` come from?  Seems like this shouldn't be there.

Comment (rawls238) [inline docs/04-logging.md]: I was quite puzzled about that myself :). We should just remove it or have it return a meaningful value

Comment (eytan) [inline docs/04-logging.md]: yeah it should be removed: you only need to specify a return value when short circuiting or disabling exposure logging.

**Pull request merged**